### ISSUE TITLE
Jenkins build waiting on LibGDX-mac

### ISSUE THREAD

**Issue by EtK2000** [libgdx/libgdx#5211]
The Jenkins build server has been waiting on Mac build for a few days already.
The message it gives is:
> (pending—All nodes of label ‘Mac’ are offline)

[Here it can be seen in more detail](http://144.76.220.132:8080/label/Mac/)

I don't know if this is intended, and if not I'd assume this should be checked.

**Comment by badlogic** [comment]
It will be fixed Monday

**Comment by EtK2000** [comment]
I might be wrong, but this would appear to still be an issue.

**Comment by EtK2000** [comment]
Quick question which might be unrelated.
When will the next snapshot be released?
(I assumed these two were linked, but this is fixed and I haven't seen a new snapshot in a bit)

**Comment by mgsx-dev** [comment]
@Tom-Ski jenkins is stuck, waiting for a MAC since 10 days now.  Do you think it's doable to switch from jenkins to travis-ci (which provides OSX build env) in the future? or partially move jobs requiring it to Travis?

I was about to open a new issue for this, maybe this one could be reopened?
